README.md

```diff
@@ -83,6 +83,28 @@ You can run tests locally (directly on your host), or on the docker machine.
           and after that retry running the tests: `docker exec -it BFA_local pytest`
 
 
+### Staging deployment
+
+Please ensure that file in `nginx/sites_enabled/bridge-stage.conf` exists and
+is configured in proper way.
+
+Run docker-compose up command with `docker-compose-stage.yml` file
+to start production deployment:
+
+    [sudo] docker-compose -f ./docker-compose-stage.yml up
+
+Production deployment contains three containers:
```

Comment:
Stage and prod contain five containers, local four. Please add appropriate changes to the README.